brainscore_vision/benchmarks/hebart2023/__init__.py

```diff
@@ -0,0 +1,5 @@
+from brainscore_vision import benchmark_registry
+from .benchmark import Hebart2023Accuracy
+
+benchmark_registry['Hebart2023-acc'] = Hebart2023Accuracy
```

Comment:
let's call it "match" rather than "accuracy"? I think accuracy implies to most people the top-1 accuracy with respect to ground-truth labels, whereas here it's the match with human choices right?

Reply:
I think I changed all instances.